orangecontrib/text/widgets/owldavis.py

```diff
@@ -132,10 +131,10 @@ def clear_all(self) -> None:
         self.legend.hide()
 
     def update_graph(
-        self,
-        words: List[str],
-        term_topic_freq: np.ndarray,
-        marginal_probability: np.ndarray,
+            self,
+            words: List[str],
+            term_topic_freq: np.ndarray,
+            marginal_probability: np.ndarray,
```

Comment:
This one doesn't need to be indented twice since `) -> None:` is in the new line, even if PyCharm wants it. Twice indent is, in my opinion, more suitable when `) -> None:` is in the same line as the last arguments. This indent was actually proposed by the Black code formater.

```suggestion
        self,
        words: List[str],
        term_topic_freq: np.ndarray,
        marginal_probability: np.ndarray,
```